### PR TITLE
Helm example

### PR DESCRIPTION
**Pull request by JenMadiedo**
Issue Number: AD52

## Summary
Merging example helm chart with readme for how to deploy

## Changes

- List of comprehensive changes

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [ ] Documentation Updates Made?
- [ ] This is not a breaking change. If it is, please describe it below.

## Is this a breaking change?
If yes, what workflow does this break? 

## Anything else?
Other comments, collaborators, etc.
This PR Closes Issue #

### CODE REVIEW

**Comment by gfmatthews** [comment]
Do we want to leave this in ms-internal -OR- do we want to make this a top-level sample directory (the latter is my preference)

**Comment by juzuluag** [comment]
> Do we want to leave this in ms-internal -OR- do we want to make this a top-level `samples` directory (the latter is my preference)

Agree. That was one of the items in today's standup.

**Comment by JenMadiedo** [comment]
> > Do we want to leave this in ms-internal -OR- do we want to make this a top-level `samples` directory (the latter is my preference)
> 
> Agree. That was one of the items in today's standup.
@gfmatthews made the change

File: samples/helmexample/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          livenessProbe:
```
Comment (rakker91) [inline samples/helmexample/templates/deployment.yaml:40]: Should we make liveness and readiness probes variables from values.yaml?

File: samples/helmexample/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+* The `repository` field to be `<acr-login-server>/<image-repository>`
+* The `pullPolicy` field to be `IfNotPresent` (pulls a new image if not present) or `Always` (pulls a new image every time).
+* The `tag` field if you need a particuar tag version
+
+Set the `nameOverride` and `fullNameOverride` fields to make it easier to reference your helm chart, and ensure they are not the same.
+
+In the `serviceAccount` section, ensure that
+* The `name` field is set to the name of the helm chart from `Chart.yaml`.
+
+### Deployment.yaml
```
Comment (rakker91) [inline samples/helmexample/README.md:74]: Should we make this variables from Values.Yaml?  Generally not considered to be a good thing to need to modify stuff in "templates" as a standard process.

**Review by rakker91 — APPROVED** [review]

File: samples/helmexample/templates/deployment.yaml (hunk 1 above)
Comment (JenMadiedo) [reply samples/helmexample/templates/deployment.yaml:40]: Made that change, thanks for the suggestion

File: samples/helmexample/README.md (hunk 2 above)
Comment (JenMadiedo) [reply samples/helmexample/README.md]: Updates this and the readme accordingly, thx

**Review by rakker91 — APPROVED** [review]

**Comment by gfmatthews** [comment]
Note: declining this review as I'm not the right person to review anyone's helm charts :)

File: samples/helmexample/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+# How-To: Deploy to AKS using Helm
+
+## Setup Resource Group
```
Comment (gfmatthews) [inline samples/helmexample/README.md:3]: Non-blocking idea: Add the az CLI options or download the arm template to create such an environment

**Review by gfmatthews — COMMENTED** [review]
Readme looks solid - I comment as a bystander :)

File: samples/helmexample/README.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+* List all deployments in all namespaces: `kubectl get deployments --all-namespaces=true`
+* List details about a specific deployment: `kubectl describe deployment <deployment-name> --namespace <namespace-name>`
+* Delete a specific deployment: `kubectl delete deployment <deployment-name>`
+* List pods: `kubectl get pods`
+* Check on a specific pod: `kubectl describe pod <pod-name>`
+* Get logs on a specific pod: `kubectl logs <pod-name>`
+* Delete a specific pod: `kubectl delete pod <pod-name>`
+
+## References
+Helm 3 docs - https://helm.sh/docs/
```
Comment (bderusha) [inline samples/helmexample/README.md:127]:
```suggestion
Helm 3 docs - https://helm.sh/docs/

```

**Review by bderusha — APPROVED** [review]
Looks good!  Just a few small comments to take or leave

**Pull request merged**